app/assets/stylesheets/components/_container.scss

```diff
@@ -19,7 +19,6 @@
   .btn-primary {
     background-color: #f57e21;
     border-color: #914810;
-    margin: 20px 0px;
```

Comment:
This style on one button was giving the whole button bar more height, but it didn't align with the new column added in this PR.